vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -23537,6 +23557,25 @@ unit:M3-PER-SEC2
   rdfs:label "Cubic Meter per Square Second"@en-us ;
   rdfs:label "Cubic Metre per Square Second"@en ;
 .
+unit:M3-PER-YR
+    a qudt:Unit ;
+    dcterms:description "power of the SI base unit metre with the exponent 3 divided by the unit year"^^rdf:HTML ;
+    qudt:applicableSystem sou:CGS ;
+    qudt:applicableSystem sou:CGS-EMU ;
+    qudt:applicableSystem sou:CGS-GAUSS ;
+    qudt:applicableSystem sou:SI ;
+    qudt:conversionMultiplier 3.168808781e-8 ;
```

Comment:
Should be 
qudt:conversionMultiplier 0.00000003168808781 ;
qudt:conversionMultiplierSN 3.168808781E-8 ;